gt/main.c

```diff
@@ -842,11 +842,14 @@ print_unsent_policy(struct ggu_policy *policy,
 
 		ret = snprintf(err_msg, sizeof(err_msg),
 			"gt: failed to send out the notification to Gatekeeper with policy decision [state: GK_BPF (%hhu), expire_sec: %u, program_index=%u, cookie="
```

Comment:
Sorry for being so picky, but let's make `expire_sec` the same as in `print_flow_state()`, that is `0x%"PRIx64"`

Reply:
No problem. I prefer small fixes. Although note that `expire_sec` is a 32-bit number here because it's a number of seconds, whereas `expire_at` (in the GK block) is a 64-bit number because it is measured in cycles. Just rebased with this change.

Reply:
I was blind to this difference. In this case, we should have `%u` for `expire_sec` in order to make it easier to read.

Reply:
OK, I just put it back to the original.

Reply:
Thank you.